services/sr_asset_assign.py

```diff
@@ -113,45 +125,6 @@ def get_params(layer_config, point, token):
     return params
 
 
-def handle_features(layer, record, connected_field, res):
```

Comment:
we used to have to make this round trip back to Knack to query out an AGOL record's Knack ID, but those IDs have been available in AGOL layers since we switched to knack services 😅

Reply:
💯  much easier now